### ISSUE TITLE
Black should remove empty parentheses after class names

### ISSUE THREAD

**Issue by ambv** [ambv/black#145]
```
class SomeClass():
    ...
```
should be just
```
class SomeClass:
    ....
```

Those are equivalent both on Python 3 and Python 2.

**Comment by skapil-chegg** [comment]
@ambv can I take this one?

**Comment by skapil** [comment]
@ambv can I take this one?

**Comment by ambv** [comment]
Go for it!

**Comment by ambv** [comment]
@skapil Let me know if anything is unclear in the codebase. Happy to guide you through it!

**Comment by skapil** [comment]
Thanks @ambv. I am still getting trying to understand the code and looks like code making sense for me slowly. However, I would definitely reach out if I would have any question!!

**Comment by csurfer** [comment]
@ambv : Should we remove the braces or would it be better to replace it with
```python
class SomeClass():
    ...

# to

class SomeClass(object):
    ...
```

Wouldn't inheriting object be more pythonic? How costly would it be in terms of perf?

**Comment by cclauss** [comment]
I like the latter under the rule _it is better to be explicit than implicit__.

**Comment by zsol** [comment]
In python2 those things are not equivalent

**Comment by cclauss** [comment]
Agreed but at this stage (Python 2.7) what are the downsides of autoswitching to [New Style Classes](https://www.python.org/doc/newstyle).  I know that on earlier versions of Python it might have been a problem to autoswitch but in current Python 2, are there still problems in moving to New Style Class?

**Comment by csurfer** [comment]
Fair enough! Just wanted to start a discussion to ensure it is thought about as I didn't see that possibility being discussed.

**Comment by skapil** [comment]
@ambv Can you please review https://github.com/ambv/black/pull/180. Working on tests right now.

**Comment by ambv** [comment]
This is fixed by #180. Thanks, @skapil! ✨ 🍰 ✨
